matchzoo/engine/base_preprocessor.py

```diff
@@ -57,8 +57,8 @@ def fit_transform(self, inputs: list, stage: str) -> datapack.DataPack:
 
     def _make_output(
         self,
-        output: list,
-        mapping: pd.DataFrame,
+        output: pd.DataFrame,
```

Comment:
`output` is `data` defined in `datapack`, while `data` is defined as `list` or `ndarray`, output is `pd.DataFrame`.

Reply:
I have mis-understand the output. It has been revised in the next commit.